index.js

```diff
@@ -0,0 +1,25 @@
+
+const openNavButton = document.getElementById("open-nav-button");
+
+openNavButton.addEventListener("click", () => {
+  document.getElementById("mySidepanel").style.width = "200px";
+});
+
+const closeNavButton = document.getElementById("close-nav-button");
+
+closeNavButton.addEventListener("click", () => {
+  document.getElementById("mySidepanel").style.width = "0";
+});
+
+const tasks = [];
+const addTaskButton = document.getElementById("add-task-button");
+const taskTitleInput = document.getElementById("task-title-input");
+
+addTaskButton.addEventListener("click", () => {
+  tasks.push({
+    id: tasks.length + 1,
+    title: taskTitleInput.value,
+  });
+
+  console.log(tasks);
+});
```

Comment:
```suggestion
const sideMenuContainer = document.getElementById("side-menu-container");
const openNavButton = document.getElementById("open-nav-button");
const closeNavButton = document.getElementById("close-nav-button");
const addTaskButton = document.getElementById("add-task-button");
const taskTitleInput = document.getElementById("task-title-input");
const tasks = [];

openNavButton.addEventListener("click", () => {
  sideMenuContainer.style.width = "200px";
});

closeNavButton.addEventListener("click", () => {
  sideMenuContainer.style.width = "0";
});

addTaskButton.addEventListener("click", () => {
  tasks.push({
    id: tasks.length + 1,
    title: taskTitleInput.value,
  });

  console.log(tasks);
});
```

Reply:
1. `mySideMenu` is not the best naming and it's not following the convention that we agreed on for ids. So, would be nice if you could change it to `side-menu-container` instead.
2. As you can see on my suggestion, I moved all variables definition to top.
3. There was also one empty line in the beginning of the file which I removed.

Reply:
So helpful , I'll fix them all.